### PR TITLE
Feat 59139 symfony form builder

### PR DESCRIPTION
**Pull request by florianalexandre**

### CODE REVIEW

**Comment by noveomax** [comment]
@florianalexandre 
I've reverted the way to spawn new Field entities on form submit because I can't get it working in your way.
Also edit/delete abilities added, please review.

File: bundle/Controller/AdminController.php [hunk 1]
```diff
@@ -27,28 +31,167 @@
...
         }
 
-        return $this->render('@FormBuilder/form_builder/form/new.html.twig', [
-            'form'              => $form->createView(),
-        ]);
+        return $this->render(
+            '@FormBuilder/form_builder/form/edit.html.twig',
+            [
+                'form'                => $form->createView(),
+                'title' => $translator->trans('Create new form')
```
Comment (florianalexandre) [inline bundle/Controller/AdminController.php]: handle translation in template

File: bundle/Resources/views/form_builder/form/edit.html.twig [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <div class="container">
+        <section class="user-form col-md-6 col-md-offset-3">
+            <div>
+                {{ form_start(form) }}
+                {{ form_row(form.name) }}
+                {{ form_row(form.maxSubmissions) }}
+                {{ form_row(form.fields) }}
+
+                <div class="mt-3">
+                    <button class="btn btn-primary" type="submit">{{ 'Save'|trans }}</button>
```
Comment (florianalexandre) [inline bundle/Resources/views/form_builder/form/edit.html.twig]: Add a cancel button

File: bundle/Controller/AdminController.php [hunk 3]
```diff
@@ -27,28 +31,167 @@
...
+            $em->flush();
+
+            return $this->redirectToRoute('form_builder_form_list');
+        }
+
+        return $this->render(
+            '@FormBuilder/form_builder/form/edit.html.twig',
+            [
+                'form'                => $form->createView(),
+                'title' => $translator->trans('Edit form')
```
Comment (florianalexandre) [inline bundle/Controller/AdminController.php]: handle translation in template

**Pull request merged**